pom.xml

```diff
@@ -660,26 +666,20 @@
         </executions>
       </plugin>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-assembly-plugin</artifactId>
+        <groupId>net.nicoulaj.maven.plugins</groupId>
```

Comment:
Is it the case that maven plugin(s) don't support this functionality? Perhaps a short comment here would help future folks. I'm not familiar with this plugin (net. nicoulaj) and as such a short basis for why would help. thx.

Reply:
I will add some comments, thanks for bringing attention to this.
Checksum generation was moved to deploy plugin: https://issues.apache.org/jira/browse/MINSTALL-143
Deploy plugin will only generate checksum for artifacts moved to repository. The generated tarts will not be moved to the repository.

I did not find a way to do this from the standard maven plugins. Every article / question / forum I found suggested to use Nicoulaj's plugin.

Reply:
sgtm. thanks!

Reply:
"silently giggling to myself"?? :)